server/export.js

```diff
@@ -80,6 +80,11 @@ export default async function (dir, options) {
     hotReloader: null
   }
 
+  // Static hosted sites sometimes have different url requirements
+  if (config.exportAssetPrefix) {
+    renderOpts.assetPrefix = config.exportAssetPrefix.replace(/\/$/, '')
```

Comment:
Why can't we simply use the existing `assetPrefix` option? https://github.com/zeit/next.js#cdn-support-with-asset-prefix